README.md

```diff
@@ -122,6 +122,43 @@ You can use Visual Studio Code to run and debug specific tasks related to the ML
    - **Wait for Completion:** Choose `True` if you want the pipeline to wait for completion before exiting, or `False` to allow it to run asynchronously.
 5. The output and any debugging information will be displayed in the **Debug Console** or **Integrated Terminal**, depending on the task configuration.
 
+## Build Validation Policies for Azure Repos Git
+
+### Limitation in Azure DevOps Pipelines
+
+Azure DevOps Pipelines do not support PR triggers in YAML configuration for Azure Repos, while this functionality is available for GitHub. Attempting to use the `pr:` section in YAML (example shown below) will not work for Azure Repos:
```

Comment:
Have you tried this? https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines

Reply:
Yes the note that is on that page is what this PR is for
![image](https://github.com/user-attachments/assets/f05288e2-af81-4edb-a029-0b81afee79f8)